extending/hooks.rst

```diff
@@ -3,9 +3,11 @@
 Hooks
 =====
 
+
 .. warning::
 
-    This is an **experimental** feature subject to breaking changes in future releases.
+    This is a **migrated** feature.  Please refer to the :ref:`Migration Guidelines<conan2_migration_guide>` learn about the changes.
```

Comment:
What does this "migrated" mean? The hooks in 1.X are not migrated from anywhere. Does it mean "this feature has already been migrated to 2.0 and it is available there"?

Reply:
It was not clear to me what the status was, i must have missed that in  my reset commit.

My understanding was an updated version exists in 2.0 already for uilianries PRs to docs.

I can remove it

Reply:
```suggestion
    This is an **experimental** feature subject to breaking changes in future releases.
```